fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java

```diff
@@ -94,4 +95,17 @@ protected void setUpJMSInfo(final UriInfo uriInfo, final HttpHeaders headers) {
         }
     }
 
+    /**
+     * Produce a baseURL for JMS events using the system property fcrepo.jms.baseUrl of the form http[s]://host[:port],
+     * if it exists.
+     *
+     * @return String the base Url
+     */
+    protected String getBaseUrlProperty() {
+        final String propBaseURL = System.getProperty("fcrepo.jms.baseUrl", "");
+        if (propBaseURL.length() > 0 && propBaseURL.startsWith("http")) {
+            return uriInfo.getBaseUriBuilder().clone().uri(propBaseURL).toString();
```

Comment:
Possibly a mistaken understanding of the UriBuilder, but my assumption from the Javadocs and the usage in [IdentiferConverter](https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraBaseResource.java#L54) was to preserve the original state. Perhaps its not needed?